packages/table/src/common/internal/directionUtils.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+import { Direction } from "../direction";
+import { IMovementDelta } from "../movementDelta";
+
+export function directionToDelta(direction: Direction): IMovementDelta {
```

Comment:
Would also like to use this in `handleFocusMove`, but not in this PR:
https://github.com/palantir/blueprint/blob/be0f870933c5e8b71d87ac248ee680973056efaa/packages/table/src/table.tsx#L1704-L1719